config/backend/deployment.yaml

```diff
@@ -45,12 +45,6 @@ spec:
                   name: backend-cm
                   key: backend.metrics.address
                   optional: true
-            - name: EPHEMERAL_METRICS_SECURE
```

Comment:
We need to fix the `EPHEMERAL_METRICS_ADDR` env var as well as it is now named as `EPHEMERAL_BACKEND_METRICS_PORT` as defined in the `cmd/backend.go` file